components/camera.js

```diff
@@ -33,7 +18,8 @@ export default class CameraComponent extends React.Component {
       setpreview: false,
       imageuploaded: null,
       removed: '',
-      phototaken: ''
+      phototaken: '',
```

Comment:
I'm trying to decide if form submission for editing profile settings is worth creating a stateful component for. Would be curious to talk to you about why you choose this as opposed to hooks in this scenario